packages/web-react/src/hooks/useSpacingStyle.ts

```diff
@@ -0,0 +1,23 @@
+import { CSSProperties } from 'react';
+import { ComponentSpacingProps } from '../types';
+
+interface SpacingCSSProperties extends CSSProperties {
+  [index: `--${string}`]: string | undefined | number;
```

Comment:
I think this interface is the same as in the `useTabStyleProps`. Can we use only one, please?